bingads_downloader/downloader.py

```diff
@@ -47,39 +47,56 @@ def download_data_sets(api_client: BingReportClient):
+    Downloads BingAds Ads performance reports by creating report objects
     for every day since config.first_date() till today
         Args:
          api_client: BingAdsApiClient
     """
     first_date = datetime.datetime.strptime(config.first_date(), '%Y-%m-%d')
     last_date = datetime.datetime.now() - datetime.timedelta(days=1)
     current_date = last_date
-    remaining_attempts = config.total_attempts_for_single_file
+    remaining_attempts = config.total_attempts_for_single_day
     while current_date >= first_date:
         print(current_date)
-        relative_filepath = Path('{date:%Y/%m/%d}/bing/'.format(
+        relative_filepath = Path('{date:%Y/%m/%d}/bing_v2/'.format(
             date=current_date))
         filepath = ensure_data_directory(relative_filepath)
 
         if not filepath.is_dir() or (last_date - current_date).days < 31:
-            report_request = build_ad_performance_request_for_single_day(api_client, current_date)
+            report_request_ad = build_ad_performance_request_for_single_day(api_client, current_date)
+            report_request_keyword = build_keyword_performance_request_for_single_day(api_client, current_date)
+            report_request_campaign = build_campaign_performance_request_for_single_day(api_client, current_date)
+
+
             with tempfile.TemporaryDirectory() as tmp_dir:
                 tmp_filepath = Path(tmp_dir, relative_filepath)
                 tmp_filepath.parent.mkdir(exist_ok=True, parents=True)
                 try:
                     start_time = time.time()
-                    submit_and_download(report_request, api_client, str(filepath))
-                    print('Successfully downloaded data for {date:%Y-%m-%d} in {elapsed:.1f} seconds'
+                    print('About to download ad data for {date:%Y-%m-%d}'
+                          .format(date=current_date))
+                    submit_and_download(report_request_ad, api_client, str(filepath), config.ad_performance_data_file())
+                    print('Successfully downloaded ad data for {date:%Y-%m-%d} in {elapsed:.1f} seconds'
                           .format(date=current_date, elapsed=time.time() - start_time))
+                    start_time = time.time()
+                    print('About to download keyword data for {date:%Y-%m-%d}'
+                          .format(date=current_date))
+                    submit_and_download(report_request_keyword, api_client, str(filepath), config.keyword_performance_data_file())
+                    print('Successfully downloaded keyword data for {date:%Y-%m-%d} in {elapsed:.1f} seconds'
+                      .format(date=current_date, elapsed=time.time() - start_time))
+                    print('About to download campaign data for {date:%Y-%m-%d}'
+                          .format(date=current_date))
+                    submit_and_download(report_request_campaign, api_client, str(filepath), config.campaign_performance_data_file())
```

Comment:
module 'bingads_downloader.config' has no attribute 'campaign_performance_data_file'

Reply:
Put back, now with the documentation and the CLI command